frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mention/mention_block.dart

```diff
@@ -42,18 +42,51 @@ Node dateMentionNode() {
 class MentionBlockKeys {
   const MentionBlockKeys._();
 
-  static const reminderId = 'reminder_id'; // ReminderID
   static const mention = 'mention';
   static const type = 'type'; // MentionType, String
+
   static const pageId = 'page_id';
   static const blockId = 'block_id';
 
   // Related to Reminder and Date blocks
   static const date = 'date'; // Start Date
   static const includeTime = 'include_time';
+  static const reminderId = 'reminder_id'; // ReminderID
   static const reminderOption = 'reminder_option';
 
   static const mentionChar = '\$';
+
+  static Map<String, dynamic> buildMentionPageAttributes({
+    required String mentionType,
```

Comment:
We can replace the `String` with `MentionType`.

Reply:
Done!